firebase.json

```diff
@@ -10,112 +10,162 @@
       "src",
       "**/node_modules/**"
     ],
+    "rewrites": [],
     "redirects": [
       { "regex": "^/(flutter(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "^/(objcdoc(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "^/(javadoc(/.*)?)", "destination": "https://api.flutter.dev/:1", "type": 301 },
       { "regex": "(?P<basename>.*)\\.html$", "destination": ":basename", "type": 301 },
       { "regex": "(?P<basename>.*)\\.$", "destination": ":basename", "type": 301 },
+
+      { "source": "/community", "destination": "https://flutter.dev/community", "type": 301 },
+      { "source": "/showcase", "destination": "https://flutter.dev/showcase", "type": 301 },
+      { "source": "/subscribe", "destination": "https://flutter.dev", "type": 301 },
```

Comment:
Will the subscribe page me moving to the marketing site? Can we remove this page?

Reply:
Can `/subscribe` be a redirect to the anchor `https://flutter.dev/#newsletter`?

Reply:
Anchor is a good idea, but I do agree that these are not necessary anymore. Removed.